internal/database/schema.sql

```diff
@@ -510,12 +510,52 @@ CREATE TABLE IF NOT EXISTS user_logins (
   user_id TEXT REFERENCES users(id),
 );
 
+CREATE SEQUENCE IF NOT EXISTS race_series_id_sequence START 1;
+
+CREATE TABLE IF NOT EXISTS race_series (
+  id                    INTEGER PRIMARY KEY DEFAULT nextval('race_series_id_sequence'),
+  name                  TEXT,
+  vehicle_id            USMALLINT REFERENCES vehicles(id),
+  vehicle_class_id      USMALLINT REFERENCES vehicle_classes(id),
+  active                BOOLEAN DEFAULT false,
+  created_at            TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  started_at            TIMESTAMP,
+  ended_at              TIMESTAMP
+);
+
+CREATE SEQUENCE IF NOT EXISTS race_events_id_sequence START 1;
+
+CREATE TABLE IF NOT EXISTS race_events (
+  id                    INTEGER PRIMARY KEY DEFAULT nextval('race_events_id_sequence'),
+  race_series_id        INTEGER REFERENCES race_series(id),
+  location_id           USMALLINT REFERENCES locations(id),
+  name                  TEXT,
+  created_at            TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  vehicle_id            USMALLINT REFERENCES vehicles(id),
```

Comment:
Since in real events drivers can drive different cars as long as they are from the same vehicle class, we should omit vehicle ID from events and series

Reply:
Removed